pytorch_lightning/core/grads.py

```diff
@@ -36,20 +38,12 @@ def grad_norm(self, norm_type: Union[float, int, str]) -> Dict[str, float]:
             norms: The dictionary of p-norms of each parameter's gradient and
                 a special entry for the total p-norm of the gradients viewed
                 as a single vector.
-        """
```

Comment:
do we usually remove the docstrings too? there is a small chance they would go out of sync between 1.3 and 1.5 who knows.

Reply:
Yes, let's remove this docstring